pkg/endpoints/dashboard.go

```diff
@@ -107,9 +107,9 @@ func IsOpenShift(r Resource, installedNamespace string) bool {
 }
 
 // Get pipelines version
-func GetPipelineVersion(r Resource, isOpenShift bool) string {
+func GetPipelineVersion(r Resource, installedNamespace string, isOpenShift bool) string {
 	version := ""
-	namespacesToCheck := []string{"tekton-pipelines"}
+	namespacesToCheck := []string{installedNamespace}
```

Comment:
Tekton Pipeline release yaml is always installed into `tekton-pipelines` except when using openshift operator where it is installed in `openshift-pipelines` so is the most likely place to find it installed
If the dashboard is installed into a different namespace then this wouldn't pick up the pipeline version on the About page as it would then look for Pipeline in the installed namespace of the dashboard and would surface errors

Reply:
I know this is not perfect yet, that's why i added a note in the PR description, i thought about creating another PR to address this issue.

It could be improved by adding related environment variables i guess.

As i am working on adding Tekton helm charts, the target namespace can now be configured at release install time.